autopr/database.py

```diff
@@ -38,8 +38,10 @@ class Database:
         default_factory=list
     )  # is equal to assigning []
 
-    def non_removed_repositories(self) -> List[Repository]:
-        return [repo for repo in self.repositories if not repo.removed]
+    def repositories_to_process(self) -> List[Repository]:
```

Comment:
I like the name better. A comment explaining that we remove the "removed" repositories here would be good.